src/pretix/static/pretixbase/js/addressform.js

```diff
@@ -0,0 +1,63 @@
+$(function () {
+    "use strict";
+
+    $("select[data-country-information-url]").each(function () {
+        let counter = 0;
+        const dependency = $(this),
+            url = this.getAttribute('data-country-information-url'),
```

Comment:
I’d add the loading-indicator once to the DOM and just hide and show it. Maybe even fade?
```suggestion
            loader = $("<span class='fa fa-cog fa-spin'></span>").hide().prependTo(dependency.closest(".form-group").find("label")),
            url = this.getAttribute('data-country-information-url'),
```